codersdk/deployment.go

```diff
@@ -1215,10 +1216,20 @@ when required by your organization's security policy.`,
 			YAML:        "stackdriverPath",
 			Annotations: clibase.Annotations{}.Mark(annotationExternalProxies, "true"),
 		},
+		{
+			Name:        "Enable Terraform debug mode",
+			Description: "Enable debug mode while processing Terraform templates.",
```

Comment:
```suggestion
			Description: "Allow administrators to enable debug mode while processing Terraform templates.",
```

> while processing Terraform templates.

Does it only apply to templates? Or is it workspace builds as well? Maybe something more generic like "Allow administrators to enable Terraform debug output."?

Reply:
Gotcha! I changed to the more generic form 👍